docs/source/topologies_beaker.rst

```diff
@@ -0,0 +1,36 @@
+Beaker Topologies
+=================
+
+.. contents:: Topics
+
+.. _beaker_topologies:
+
+
+Beaker Server
+```````````````
+
+.. code-block:: yaml
+
+    ---
+    topology_name: beaker
+    resource_groups:
+      - resource_group_name: test1
+        res_group_type: beaker
+        job_group: ci-ops-central
+        recipesets:
+          - distro: RHEL-6.5
+            arch: x86_64
+            arches:
+              - X86_64
+              - X86_64
+            keyvalue:
+              - MEMORY>1000
+              - DISKSPACE>20000
+            hostrequire:
+              - arch=X86_64
+    count: 1
```

Comment:
I think this was a copypasta error; the indentation is lost when copying from the GitHub web interface.  Nevertheless, I did run a test to verify that within the **recipesets** section is the correct place to put the **count** setting.  Here's a test run where I requested two Beaker servers and successfully provisioned and then dropped them:

https://gist.github.com/robled/23a1f844aeb811f1e5309906c5aa07a3